io.catenax.shared.message_header/2.0.0/MessageHeaderAspect.ttl

```diff
@@ -0,0 +1,140 @@
+   samm:characteristic :ContextCharacteristic;
+   samm:exampleValue "urn:samm:io.catenax.<ASPECT-MODEL-NAME>:1.x.x".
+
+:sentDateTime a samm:Property;
+   samm:preferredName "Sent Date Time"@en;
+   samm:description "Time zone aware timestamp holding the date and the time the message was sent by the sending party. The value MUST be formatted according to the ISO 8601 standard"@en;
+   samm:characteristic samm-c:Timestamp;
+   samm:exampleValue "2023-06-19T21:24:00+07:00"^^xsd:dateTime.
+
+:senderBpn a samm:Property;
+   samm:preferredName "Sender BPN"@en;
+   samm:description "The Business Partner Number of the sending party. The value MUST be a valid BPN. BPNA and BPNS are not allowed. Applicable constraints are defined in the corresponding standard"@en;
+   samm:characteristic :BpnCharacteristic;
+   samm:exampleValue "BPNL7588787849VQ".
+
+:receiverBpn a samm:Property;
+   samm:preferredName "Receiver BPN"@en;
+   samm:description "The Business Partner Number of the receiving party. The value MUST be a valid BPN. BPNA and BPNS are not allowed. Applicable constraints are defined in the corresponding standard."@en;
+   samm:characteristic :BpnCharacteristic;
+   samm:exampleValue "BPNL6666787765VQ".
+
+:expectedResponseBy a samm:Property;
+   samm:preferredName "Expected Response By"@en;
+   samm:description "Time zone aware timestamp holding the date and time by which the sending party expects a certain type of response from the receiving party. The meaning and interpretation of the fields's value are context-bound and MUST therefore be defined by any business domain or platform capability making use of it. The value MUST be formatted according to the ISO 8601 standard"@en;
+   samm:characteristic samm-c:Timestamp;
+   samm:exampleValue "2023-06-19T21:24:00+07:00"^^xsd:dateTime.
+
+:relatedMessageId a samm:Property;
+   samm:preferredName "Related Message ID"@en;
+   samm:description "Unique ID identifying a message somehow related to the current one"@en;
+   samm:characteristic :IdTrait;
+   samm:exampleValue "d9452f24-3bf3-4134-b3eb-68858f1b2362".
+
+:version a samm:Property;
+   samm:preferredName "Version"@en;
+   samm:description "The unique identifier of the aspect model defining the structure and the semantics of the message's header. The version number should reflect the versioning schema of aspect models in Catena-X."@en;
+   samm:characteristic :SemanticVersioningTrait;
+   samm:exampleValue "urn:samm:io.catenax.message_header:2.0.0".
+
+:IdTrait a samm-c:Trait;
+   samm:preferredName "ID (Identifier) Trait"@en;
+   samm:description "Trait for defining the UuidCharacteristic to be a UUIDv4 compliant property."@en;
+   samm-c:baseCharacteristic :UuidCharacteristic;
```

Comment:
Also, let's use shared model for UUID Characteristic

Reply:
Done :)